src/index.css

```diff
@@ -161,6 +159,40 @@ nav a.active-link {
   display: none;
 }
 
+.terms-and-about {
+  margin-left: 16px;
+  margin-right: 16px;
+
+  p {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  h4 {
+    font-style: italic;
+    font-size: 14px;
+    margin-bottom: 32px;
+  }
```

Comment:
You're styling this italic because the "Last updated at" line is italic. However, this way any `h4` would be italic. I know that at the moment there's none, but still, it would be more specific if you used a class name instead of an element selector (like, `.last-updated` or `.post-metadata` or the like).